test/e2e-secondary-network/framework.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	antreae2e "antrea.io/antrea/test/e2e"
+	corev1 "k8s.io/api/core/v1"
+	"log"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+type TestData struct {
```

Comment:
After checking the code, I do not see why we need to redefine TestData, rather than reusing `antreae2e.TestData`, then we can reuse all util funcs of  `antreae2e.TestData`, rather than redefining them too.

The only field you added to TestData is `logsDirForTestCase`, but I feel that field is not needed at all if we reuse `antreae2e.TestData`.